js/src/util.js

```diff
@@ -93,17 +93,21 @@ const Util = {
 
     // Get transition-duration of the element
     let transitionDuration = $(element).css('transition-duration')
+    let transitionDelay = $(element).css('transition-delay')
+
     const floatTransitionDuration = parseFloat(transitionDuration)
+    const floatTransitionDelay = parseFloat(transitionDelay)
 
     // Return 0 if element or transition duration is not found
-    if (!floatTransitionDuration) {
+    if (!floatTransitionDuration && !floatTransitionDelay) {
       return 0
     }
 
     // If multiple durations are defined, take the first
     transitionDuration = transitionDuration.split(',')[0]
+    transitionDelay = transitionDelay.split(',')[0]
 
-    return parseFloat(transitionDuration) * MILLISECONDS_MULTIPLIER
+    return (parseFloat(transitionDuration) + parseFloat(transitionDelay)) * MILLISECONDS_MULTIPLIER
```

Comment:
I wonder, can't we combine this with the same effect? Like `parseFloat(transitionDuration + transitionDelay)`

Reply:
Don't think so. `transitionDuration` and `transitionDelay` are stings like `0.6s`.

Reply:
Please try it because it should automatically be coerced.

Reply:
I don't think so 😟 
![image](https://user-images.githubusercontent.com/1689750/47654861-0aecde00-db8c-11e8-94fd-fba925536555.png)

Reply:
Ok, thanks for confirming :)